spec/services/calculators/income_contribution_calculator_spec.rb

```diff
@@ -13,6 +13,14 @@ module Calculators
         end
       end
 
+      context "negative income" do
+        let(:income) { -100.0 }
+
+        it "returns zero" do
+          expect(calculator).to be_zero
+        end
```

Comment:
nit
```suggestion
        it { expect(calculator).to be_zero }
```

Reply:
Thanks for the suggestion @jsugarman. I think I prefer to keep the it block for the description in this case and it is maybe more consistent with the other tests in this spec.